src/hidimstat/utils.py

```diff
@@ -9,6 +9,7 @@
 from torchmetrics import Accuracy
 
 
+########################## quantile aggregation method ##########################
```

Comment:
In the long term, some stats routine should go to a `src/hidimstat/statistics` module